openpathsampling/netcdfplus/cache.py

```diff
@@ -374,7 +374,7 @@ class LRUChunkLoadingCache(Cache):
 
     """
 
-    def __init__(self, chunksize=100, max_chunks=100, variable=None):
+    def __init__(self, chunksize=256, max_chunks=8192, variable=None):
```

Comment:
There is no testing. I tried to use number of base 2. I do not have time at this point to run an optimisation. I am happy that I find time to work at all on this. This is based on caching about 2M values and chunk size of 256 to match number I used for the other objects.

I think we should name the optimization 1.X